### ISSUE TITLE
The download builder is broken

### ISSUE THREAD

**Issue by teo1978** [jquery/jquery-mobile#8657]
Steps to reproduce:

1. Go to https://jquerymobile.com/download/

The first thing it says is "we **RECOMMEND** using our Download Builder (links to it)...", so:

2. Follow the link to https://jquerymobile.com/download-builder/

Expected:
Should be able to use the fucking download builder

Observed:

![image](https://user-images.githubusercontent.com/542107/65696325-526c3a80-e079-11e9-9309-9c924d600bb9.png)

The spinning icon keeps spinning forever and there's nothing you can build *or* download.

I think this JS error in the console might be the culprit:
```
jquery-1.11.3.js:9664 GET https://amd-builder.jquerymobile.com/v1/dependencies/jquery/jquery-mobile/1.4.5/?baseUrl=js 502 (Bad Gateway)
send @ jquery-1.11.3.js:9664
ajax @ jquery-1.11.3.js:9215
jQuery.<computed> @ jquery-1.11.3.js:9361
getJSON @ jquery-1.11.3.js:9344
refreshForm @ builder.js:149
(anonymous) @ builder.js:185
fire @ jquery-1.11.3.js:3148
fireWith @ jquery-1.11.3.js:3260
ready @ jquery-1.11.3.js:3472
completed @ jquery-1.11.3.js:3503
(index):1 Access to XMLHttpRequest at 'https://amd-builder.jquerymobile.com/v1/dependencies/jquery/jquery-mobile/1.4.5/?baseUrl=js' from origin 'https://jquerymobile.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by FritzTheBlitz** [comment]
seams to be a duplicate of #8654
